page/ajax/ajax-events.md

```diff
@@ -5,7 +5,8 @@
 	"attribution": [ "jQuery Fundamentals" ]
 }</script>
 
-Often, you'll want to perform an operation whenever an Ajax request starts or stops, such as showing or hiding a loading indicator. Rather than defining this behavior inside every Ajax request, you can bind Ajax events to elements just like you'd bind other events. For a complete list of Ajax events, visit [Ajax Events documentation on docs.jquery.com](http://docs.jquery.com/Ajax_Events).
+
```

Comment:
Can you remove the addition of a new blank line in here? They're unrelated changes and there being 2 lines would be inconsistent.